### PR TITLE
Implementation of admin panel

### PR DESCRIPTION
**Pull request by almerion**

### CODE REVIEW

File: src/ajax/delete_element.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+if (isset($_POST["id"], $_POST["mode"]) && $_SESSION["role"] == 1) {
+    $id = $_POST["id"];
+    $mode = $_POST["mode"];
+    $query = "";
+
+    if ($mode == 0)
+        $query = deleteAccount();
+    else if ($mode == 1)
+        $query = deleteImage();
```
Comment (NaulaN) [inline src/ajax/delete_element.php:15]:
```suggestion
    $query = ($mode == 0) ? deleteAccount() : deleteImage();
```
This is like a boolean condition, you can write this condition with only one line

File: src/modules/mod_admin_panel/cont_admin_panel.php [hunk 2]
```diff
@@ -17,4 +17,22 @@ public function displayMod()
...
+    public function getAdminPannel($mode)
+    {
+        $datas = null;
+        switch ($mode) {
+            case 0:
+                $datas = $this->model->getAccounts();
+                break;
+            /*case 1:
+                $datas = $this->model->get();
+                break;*/
```
Comment (NaulaN) [inline src/modules/mod_admin_panel/cont_admin_panel.php:30]: Dirty code for merging into `master` branch

File: src/modules/mod_admin_panel/cont_admin_panel.php [hunk 3]
```diff
@@ -17,4 +17,22 @@ public function displayMod()
...
+            default:
+                $datas = $this->model->getImages();
+                break;
+        }
```
Comment (NaulaN) [inline src/modules/mod_admin_panel/cont_admin_panel.php:34]:
```suggestion
        $datas = ($mode == 0) ? $this->model->getAccounts() : $this->model->getImages();
```

File: src/modules/mod_admin_panel/model_admin_panel.php [hunk 4]
```diff
@@ -15,4 +15,28 @@ public function insertImages($idcompte, $nom, $partager, $bin)
...
+        return $prepare->fetchAll(PDO::FETCH_ASSOC);
+    }
+
+    /*public function getExercises()
+    {
+        $query = "SELECT idexercice as id,exercices.nom , login, compte.nom, prenom, (SELECT 2) as mode FROM public.exercices inner join public.compte on (exercices.idcompte = compte.idcompte)";
+        $prepare = parent::$bdd->prepare($query);
+        $prepare->execute();
+        return $prepare->fetchAll();
+    }*/
```
Comment (NaulaN) [inline src/modules/mod_admin_panel/model_admin_panel.php:33]: Dirty code for merging into `master` branch

**Review by NaulaN — CHANGES_REQUESTED** [review]

File: src/ajax/delete_element.php (hunk 1 above)
Comment (almerion) [reply src/ajax/delete_element.php:15]: can have more modes so it should stay with if else

File: src/modules/mod_admin_panel/cont_admin_panel.php (hunk 3 above)
Comment (almerion) [reply src/modules/mod_admin_panel/cont_admin_panel.php:34]: same

**Comment by NaulaN** [comment]
@almerion good for merging ?

**Review by NaulaN — APPROVED** [review]

**Pull request merged**